### ISSUE TITLE
 As a user, I want to upload a pair of files via a drag-and-drop interface

### ISSUE THREAD

**Issue by theecrit** [openoakland/woeip#184]
*Description*
I have a CSV and log file from a single session, and I’d like to upload them to the server as seamlessly as possible via a drag-and-drop model.

**Note:** this issue is a counterpart to #129. Whichever is complete first should serve as the basis for the other.

*Acceptance criteria* 
- [x] Upload interface includes affordance for drag-and-drop per the attached prototype.
- [x] User can see a list of and remove selected files from the interface prior to confirming upload.
- [x] All open questions in Issue have been answered or moved into new Issues.

*Dependencies and assumptions*
- ~User must be authenticated (signed in).~ We'll add the account gateway once the front-end is in place.
- System uploads files to a server-side directory (same server that's hosting the database).
- Status notifications and progress indicators will be covered by a separate issue.
- View will be built in React.
- User can only upload one pair at a time.

*Related documentation*
- [Prototype: happy path](https://www.figma.com/proto/IZw8VvX1CsRc764CTvg4aRSU/WOAQ-Wireframes-Parent-File?node-id=340%3A5671&scaling=contain)
- [Prototype: error path example](https://www.figma.com/proto/IZw8VvX1CsRc764CTvg4aRSU/WOAQ-Wireframes-Parent-File?node-id=341%3A5663&scaling=contain) for reference only; not covered by this issue (Note: See #131 for list of error use cases)
- [WOAQ-Data-Flow-Diagram-v5.pdf](https://github.com/openoakland/woeip/files/4157171/WOAQ-Data-Flow-Diagram-v5.pdf)
- [WOAQ-Upload-v5-Wireframes.pdf](https://github.com/openoakland/woeip/files/3937179/WOAQ-Upload-v5-Wireframes.pdf)
- ~[WOAQ-Upload-v4](https://drive.google.com/open?id=1Al5ILibeITvFcentrr0ia9R5QLRs9pIq)~

*Questions?*
- For back-end questions, reach out to @jayqi
- For visual design questions, reach out to Niels or Kimberly Low in #WOAQ Slack channel.
- For UX/flow, content, or anything else, reach out to @theecrit
